specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt

```diff
@@ -36,22 +36,20 @@ write close
 read status "200" /.+/
 read header "Content-Type" "application/octet-stream"
 read header "Connection" "close"
+read notify DOWNSTREAM_OPENED
 
-read [0x80 0x82 0x00]
-read [0..256]
-read notify DATA_RECEIVED
```

Comment:
Why did you remove this? Just because it's unnecessary? (which is a fine reason, just wanted to know if there was any other)